tests/create_module_query_log_kms/main.tf

```diff
@@ -9,8 +9,6 @@ module "query_log" {
     aws = aws
   }
 
-  create_route53_query_log = true
```

Comment:
need to add this back now, or the query_log test won't actually create a query log...